flink-runtime/src/main/java/org/apache/flink/runtime/io/network/netty/NettyProtocol.java

```diff
@@ -120,10 +120,14 @@
 	 * @return channel handlers
 	 */
 	public ChannelHandler[] getClientChannelHandlers() {
-		return new ChannelHandler[] {
-			messageEncoder,
-			new NettyMessage.NettyMessageDecoder(),
-			new CreditBasedPartitionRequestClientHandler()};
+			CreditBasedPartitionRequestClientHandler networkClientHandler = new CreditBasedPartitionRequestClientHandler();
```

Comment:
CreditBasedPartitionRequestClientHandler ->NetworkClientHandler

Reply:
Modified the type.